src/Evolution/Systems/ScalarWave/EnergyDensity.hpp

```diff
@@ -35,5 +36,21 @@ template <size_t SpatialDim>
 Scalar<DataVector> energy_density(
     const Scalar<DataVector>& pi,
     const tnsr::i<DataVector, SpatialDim, Frame::Inertial>& phi) noexcept;
+
+namespace Tags {
+template <size_t SpatialDim>
+struct EnergyDensityCompute : EnergyDensity<SpatialDim>, db::ComputeTag {
```

Comment:
need to `#include "DataStructures/DataBox/Tag.hpp` for `db::ComputeTag`

Reply:
This got lost in the shifting of changes between commits.